src/App.js

```diff
@@ -1,19 +1,19 @@
-
-import React from "react";
+import React from 'react';
 import Button from '@material-ui/core/Button';
+import QaIndex from './components/qa/QaIndex';
+
 
 class App extends React.Component {
   render() {
     const { name } = this.props;
     return (
       <>
```

Comment:
You shouldn't have needed to do this- master was updated near the start of the project to have all imports.